aspnetcore/blazor/hybrid/tutorials/maui.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Build a .NET MAUI Blazor app
+author: guardrex
+description: Build a .NET MAUI Blazor app step-by-step.
+monikerRange: '>= aspnetcore-6.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 02/12/2022
+no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: blazor/hybrid/tutorials/maui
+---
+# Build a .NET MAUI Blazor app
+
+This tutorial shows you how to build and modify a .NET MAUI Blazor app. You learn how to:
+
+> [!div class="checklist"]
+> * Create a .NET MAUI Blazor app project
+> * Run the project in the Windows Machine emulator
+> * Run the project in the Android emulator
+
+At the end of this tutorial, you'll have a working app.
+
+## Prerequisites
+
+* [Supported platforms (.NET MAUI documentation)](/dotnet/maui/supported-platforms)
+* [Visual Studio 2022 Preview](https://visualstudio.microsoft.com/vs/preview/) with the **Mobile development with .NET** workload
+* [Microsoft Edge WebView2](https://developer.microsoft.com/microsoft-edge/webview2/): WebView2 is required on Windows when running a native app. When developing .NET MAUI Blazor apps and only running them in Visual Studio's emulators, WebView2 isn't required.
```

Comment:
Let's add an additional bullet point here about enabling hardware acceleration for the Android emulator:

```suggestion
* [Microsoft Edge WebView2](https://developer.microsoft.com/microsoft-edge/webview2/): WebView2 is required on Windows when running a native app. When developing .NET MAUI Blazor apps and only running them in Visual Studio's emulators, WebView2 isn't required.
* [Enable hardware acceleration](https://docs.microsoft.com/xamarin/android/get-started/installation/android-emulator/hardware-acceleration?pivots=windows) to improve the performance of the Android emulator.
```